src/module/gui/__init__.py

```diff
@@ -139,6 +140,10 @@ def set_icon(self, img: QImage|None):
 			self.icon.fill(QColor(0, 0, 0, 0))
 			self.iconButton.setIcon(self.icon)
 		print('Icon updated!')
+		if self.path is not None:
+			self.iconButton.setToolTip(f'{os.path.basename(self.path)}  Right-click to remove')
+		else:
+			self.iconButton.setToolTip('')
```

Comment:
(whoops, forgot to say this in round 1) This should actually probably all be in `self.reload`. `set_icon` is only called after the image is loaded in, so if the image fails to load then users will never get the 'right click to remove' hint.

```py
def reload(self):
	self.iconButton.setToolTip(f'{self.path.name} Right-click to remove' if self.path else '')
	self.path_box.setText(self.path.name if self.path else '')
	self.picked.emit(self.kind, self.path, self.set_icon)
	self.update_required()
```